ci-operator/step-registry/cucushift/installer/check/vsphere/disk/cucushift-installer-check-vsphere-disk-commands.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+source "${SHARED_DIR}/govc.sh"
+source "${SHARED_DIR}/vsphere_context.sh"
+export KUBECONFIG=${SHARED_DIR}/kubeconfig
+declare vsphere_datacenter
+declare vsphere_datastore
+INFRA_ID=$(jq -r .infraID ${SHARED_DIR}/metadata.json)
+check_result=0
+
+function check_disk_type() {
+    local ret=0
+    mapfile -t  vm_lists < <(govc ls /${vsphere_datacenter}/vm/${INFRA_ID})
+    
+    for vm in "${vm_lists[@]}"; do
+        disk_file=$(govc vm.info -json /${vsphere_datacenter}/vm/${INFRA_ID}/${vm} | jq -r '.VirtualMachines[].Layout.Disk[].DiskFile[]' | awk  '{print $2}' | head -n1)
+	vm_disk_type=$(govc datastore.disk.info -ds /${vsphere_datacenter}/datastore/${vsphere_datastore} ${disk_file} | grep Type | awk -F ':    '  '{print $2}')
+	if [[ -n ${DISK_TYPE} ]];then
```

Comment:
When `DISK_TYPE` is set to `thin`, the logic will throw error.

So when `DISK_TYPE` is empty, the default type is `thin`, so we can introduce a new var `disk_type`
```
disk_type="${DISK_TYPE:-thin}"
```